packages/patrol_cli/lib/src/features/drive/flutter_driver.dart

```diff
@@ -49,22 +29,41 @@ class FlutterDriver {
     _disposeScope.disposedBy(parentDisposeScope);
   }
 
+  late String driver;
```

Comment:
I dont like this `late` since there is no guarantee it was initialized (other than "I promise I checked all paths")

Reply:
I agree, but I didn't know how to do it better.

I have to be able to mock the `FlutterDriver` that `DriveCommand` uses. I can achieve this by passing an instance of it to `DriveCommand`.

`FlutterDriver` needs some information (`host`, `port`, `driver`, etc.) that is not known at the time of `DriveCommand` construction, so I have to pass that data at a later point. Previously this later point was `FlutterDriver.run()`, but I prefer that method accept only `String target` and `Device device`, that is: data unique to each invocation of it.

That's why I decided to have an `init()` method.

I'd appreciate if you looked at the code, maybe you'd have a better idea on how clean this up.